deploy/lib/Help.rb

```diff
@@ -398,30 +398,27 @@ def self.corb
     <<-DOC.strip_heredoc
       Usage: ml {env} corb [options]
 
-      See: http://marklogic.github.com/corb/index.html
+      Runs CoRB2 with given command-line options against the selected environment.
+      CoRB2 options may be specified on the commandline using either -- or -D prefix.
 
-      Required options:
-        --modules=/path/to/modules.xqy  # the xquery module to process the data
+      For example, the OPTIONS-FILE option can be specified as:
+        --options-file=/path/to/options.properties
+        -DOPTIONS-FILE=/path/to/options.properties
 
-        (Only one of the following is required)
-        --collection=collection-name    # the name of a collection to process
-        --uris=/path/to/uris-module.xqy # path to a uris module
+      CoRB2 supports options files natively using the --OPTIONS-FILE parameter.
 
-      Corb Options:
-        --threads=1                     # the thread count to use
-        --root=/                        # the root of the modules database
-        --install=false                 # whether or not to install (default: false)
+      See: https://github.com/marklogic/corb2
 
       General options:
-        -v, [--verbose]  # Verbose output
```

Comment:
I think mentioning -v could be useful, particularly with the logger.debug below..